### ISSUE TITLE
Need ability to hide certain types

### ISSUE THREAD

**Issue by yogeshkodei** [graknlabs/grakn#5676]
Please replace every line in curly brackets { like this } with appropriate answers, and remove this line.

## Problem to Solve

Certain types need to be hidden from user queries - for not giving out our IP as well as preventing common users from looking at those types

## Current Workaround

Has to be filtered out in the application layer.

## Proposed Solution

Provide a flag on the type to reflect whether it should be hidden. When hidden should be available only to the admin/IT people and not for regular application users.

## Additional Information

{ Any additional information, including logs or screenshots if you have any. }

**Comment by alexjpwalker** [comment]
Indeed, Grakn doesn't currently have the notion of _permissions_, which is essentially what we need in order to restrict access to certain queries or certain types.

There's definitely a strong case for implementing this in the future. In the meantime, as noted, the current workaround is to implement security in the application layer.
